hadoop-ozone/ozone-recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestContainerKeyService.java

```diff
@@ -176,6 +192,10 @@ public void testGetKeysForContainer() throws Exception {
     ContainerKeyMapperTask containerKeyMapperTask = new ContainerKeyMapperTask(
         ozoneManagerServiceProvider, containerDbServiceProvider);
     containerKeyMapperTask.run();
+  }
+
+  @Test
+  public void testGetKeysForContainer() throws Exception {
```

Comment:
No need to throws Exception here, as we never throw an exception from this method.


Exception 'java.lang.Exception' is never thrown in the method less... (⌘F1) 
Inspection info: Reports exceptions that are declared in a method's signature but never thrown by the method itself or its implementations/derivatives

Reply:
Sure, will fix it in a future patch alongside other changes.

Reply:
Do you mean another jira?

Reply:
If there are no findbug issues or checkstyle issues, will fix it along with another jira

Reply:
Okay..